### PR TITLE
Upgrade Keda version from 2.7.1 to 2.8.0

### PR DESCRIPTION
**Pull request by lzianekhodja**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,7 +8,15 @@ Added
 -
 
 * Add static partitioning in ArmoniK
-* Add the parameters `InitWorker__WorkerCheckRetries` and `InitWorker__WorkerCheckDelay` in the configmap of ArmoniK polling agent
+* Add the environment variable `Amqp__MaxRetries` in the configmap of ArmoniK core and Metrics exporter
+* Add the environment variables `ControlPlane__Partitions` and `ControlPlane__DefaultPartition` in the configmap of
+  ArmoniK control plane
+* Add the environment variable `ComputePlan__PartitionId` in the of ArmoniK polling agent container
```
Comment (aneojgurhem) [inline CHANGELOG.md:14]: Not linked to keda and not really true

File: infrastructure/modules/armonik/polling-agent-configmap.tf [hunk 2]
```diff
@@ -15,8 +15,6 @@ resource "kubernetes_config_map" "polling_agent_config" {
     InitWorker__WorkerCheckDelay         = "00:00:10" # TODO: make it a variable
     target_grpc_sockets_path             = "/cache"
     target_data_path                     = "/data"
-    InitWorker__WorkerCheckRetries       = "10"
-    InitWorker__WorkerCheckDelay         = "00:00:10"
```
Comment (aneojgurhem) [inline infrastructure/modules/armonik/polling-agent-configmap.tf:19]: Polling agent needs them

Comment (lzianekhodja) [reply infrastructure/modules/armonik/polling-agent-configmap.tf:19]: they were in duplicate in the configmap

File: CHANGELOG.md (hunk 1 above)
Comment (lzianekhodja) [reply CHANGELOG.md:14]: They will be updated/deleted when we will change the partitioning integration

**Review by aneojgurhem — APPROVED** [review]

**Pull request merged**